src/components/UserField/UserField.js

```diff
@@ -1,24 +1,46 @@
 import React from 'react';
-import './userField.css'
+import './userField.css';
 
 export default class UserField extends React.Component {
     constructor() {
         super();
     }
-    
     render() {
         const { users } = this.props;
         return (
             <ul>
                 {users.map((item, index) => {
                     return (
```

Comment:
if you will not write any logic in your map function don't write return and brackets use this syntax`
blabla.map((item, index) => (
  <Component 
  
  />
))